EarlyBinding/src/EXPRESS/EXPRESSOptionalImpl.h

```diff
@@ -16,6 +16,10 @@
 	along with this program. If not, see <http://www.gnu.org/licenses/>.
 */
 
+#pragma once
+#ifndef OpenInfraPlatform_EarlyBinding_EXPRESSOptional_76c70f6e_bab2_48da_b2bc_9434b6adf3dc_g
+#define OpenInfraPlatform_EarlyBinding_EXPRESSOptional_76c70f6e_bab2_48da_b2bc_9434b6adf3dc_g
+
```

Comment:
Include guard is maybe the same as is EXPRESSOptional? This might cause some problems I can imagine.

Reply:
These have been manually manipulated to have the last letter different. Should be fine.

Reply:
resolved in 7a9a70c